spec/unit/typesafe/config/concatenation_spec.rb

```diff
@@ -45,4 +45,383 @@
     expect(e.message).to include("null")
     expect(e.message).to include('{"x":"y"}')
   end
+
+  it "no arrays in string concat" do
```

Comment:
This is a nitpick, so feel free to ignore it, but I think in rspec it's conventional to format these strings like `it "should ...." do` to make things more readable. We definitely don't need to change that now though